README.rst

```diff
@@ -1,8 +1,17 @@
-Introduction
-============
+FastKML
+=======
 
 .. inclusion-marker-do-not-remove
```

Comment:
I think the inclusion marker should be moved down, this will break the rtfd documentation.